FreeRTOS-Plus/Test/CBMC/proofs/ARP/ulARPRemoveCacheEntryByMac/Makefile.json

```diff
@@ -0,0 +1,25 @@
+{
+  "ENTRY": "ulARPRemoveCacheEntryByMac",
+  "NUMBER_OF_ENTRIES":1,
```

Comment:
Could you use a larger number like 3?  Maybe document the difference between NUMBER_OF_ENTRIES and CACHE_ENTRIES?

Reply:
That was a very wrongly named variable. Apologies. Now I have removed it completely because for this proof to work, we must set that macro as 1 (or basically anything non-zero).